test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart_default.host4and6dhcp.snap.txt

```diff
@@ -100,6 +100,7 @@ fi
 
 
 
+
```

Comment:
Why are there new blank lines here?

Reply:
That should be coming from this line https://github.com/theforeman/foreman/pull/9430/files#diff-7755c8fb296da617f75e4c6e39445726b059bc099dde6151bbe5bf468dcb9f20R326 . I could drop that empty line, but then the pull provider setup block would be glued to `<%= snippet "blacklist_kernel_modules" %>`. To me a yet another blank line in the rendered template is worth clarity in the template itself

Reply:
One solution to it is including the empty line inside the `if` block, so:
```erb
<% if plugin_present?('katello') && host_param_true?('enable-remote-execution-pull') -%>
<%= save_to_file('/root/remote_execution_pull_setup.sh', snippet('remote_execution_pull_setup')) %>
chmod +x /root/remote_execution_pull_setup.sh
/root/remote_execution_pull_setup.sh

<% end -%>
<%= snippet "blacklist_kernel_modules" %>
```

And possibly using `<%- end -%>` may also help.